test/test/cpu/CpuTests.java

```diff
@@ -5,17 +5,22 @@
 
 package test.cpu;
 
-import one.profiler.test.Jvm;
+import one.convert.Arguments;
 import one.profiler.test.Output;
 import one.profiler.test.Test;
 import one.profiler.test.TestProcess;
 
 public class CpuTests {
 
-    @Test(mainClass = RegularPeak.class, enabled = false, jvmArgs = "-XX:+UseG1GC -Xmx1g -Xms1g", jvm = Jvm.HOTSPOT)
-    public void regularPeak(TestProcess p) throws Exception {
-        p.profile("-e cpu -d 3 -f %f.jfr");
-        Output out = p.readFile("%f");
-        assert out.contains("java/util/stream/SpinedBuffer\\.accept");
+    @Test(mainClass = RegularPeak.class)
+    public void regularPeakLinux(TestProcess p) throws Exception {
+        Output out = p.profile("-e cpu -d 6 -f %f.jfr");
+        String jfrOutPath = p.getFile("%f").getAbsolutePath();
+        out = out.convertJfrToCollapsed(jfrOutPath, new Arguments("-o", "collapsed", "--to", "2500"));
```

Comment:
Why do we need to specify `-o collapsed` among arguments, if the method name already suggests we are converting to collapsed? I suggest `convertJfrToCollapsed` creates `Arguments` from the passed array of strings and sets `args.output` to `collapsed` explicitly.
As a bonus, test writers won't need to call `new Arguments()` every time - they'll just pass String varargs:
```
convertJfrToCollapsed(jfrOutPath, "--to", "2500");
```